wgsl/index.bs

```diff
@@ -1125,7 +1215,7 @@ and how to use variables with it.
       <td>Read-only
       <td>Invocations in the same shader stage
       <td>[=Module scope=]
-      <td>Opaque representation of handle to a sampler or texture
+      <td>Opaque representation of handle to a [=sampler=] or [=texture=]
```

Comment:
```suggestion
      <td>Opaque representation of handle to a [=sampler=] or [=texture=] types
```

Reply:
I think this one doesn't work grammatically because of the singular "a" vs. the plural "types".
Better to replace with

       Sampler types or texture types.

and have both sampler and texture be cross-references.